oggm/tests/test_models.py

```diff
@@ -1377,6 +1377,96 @@ def test_inversion_parabolic(self):
             plt.plot(bed_shape_gl[:-3])
             plt.show()
 
+    def test_inversion_parabolic_sf_adhikari(self):
```

Comment:
Much better to define one single function and then write wrappers for different shape factor modes, which are actually called for testing and do nothing else than setting shape factor cfg and call the function doing the work. Would avoid a lot of ugly copied code